#! /bin/sh /usr/share/dpatch/dpatch-run
## 87_move_documentation.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: place html documentation under hplip-doc/HTML/

@DPATCH@
Index: hplip-3.12.4/Makefile.am
===================================================================
--- hplip-3.12.4.orig/Makefile.am	2012-05-26 10:21:29.000000000 +1000
+++ hplip-3.12.4/Makefile.am	2012-05-26 10:21:29.000000000 +1000
@@ -27,11 +27,11 @@
 
 if DOC_BUILD
 wwwsrc = doc
-www0dir = $(docdir)
+www0dir = $(hpdocdir)/HTML
 dist_www0_DATA = $(wwwsrc)/index.html $(wwwsrc)/commandline.html $(wwwsrc)/copying.html $(wwwsrc)/devicemanager.html $(wwwsrc)/faxtrouble.html $(wwwsrc)/gettinghelp.html $(wwwsrc)/hpscan.html $(wwwsrc)/mainttask.html $(wwwsrc)/plugins.html $(wwwsrc)/print.html $(wwwsrc)/printing.html $(wwwsrc)/printoptions.html $(wwwsrc)/printtroubleshooting.html $(wwwsrc)/scanning.html $(wwwsrc)/scantrouble.html $(wwwsrc)/sendfax.html $(wwwsrc)/setup.html $(wwwsrc)/systray.html $(wwwsrc)/troubleshooting.html $(wwwsrc)/uninstalling.html $(wwwsrc)/upgrading.html
-www3dir = $(docdir)/styles
+www3dir = $(www0dir)/styles
 dist_www3_DATA = $(wwwsrc)/styles/*
-www4dir = $(docdir)/images
+www4dir = $(www0dir)/images
 dist_www4_DATA = $(wwwsrc)/images/*
 endif
 
